spec/features/pets/pets_index_usability_spec.rb

```diff
@@ -0,0 +1,123 @@
+
+        click_link "Delete"
+        expect(current_path).to eq("/pets")
+        expect(page).to_not have_content("#{pet_1.name}")
+    end
+
+        it "has links to delete next to pets on the shelter pets page" do
+        shelter_1 = Shelter.create( name: "4 Paws Rescue",
+                    address: "6567 W Long Dr.",
+                    city: "Littleton",
+                    state: "CO",
+                    zip: "80123")
+
+        pet_1 = Pet.create( name:    "Teddy",
+                            image: "https://i.pinimg.com/originals/47/4e/7d/474e7d2479512428a1a4716d5d1656eb.jpg",
+                            age:    "10",
+                            sex:   "M",
+                            description: "A sweet dog looking for a loving home",
+                            shelter_id: shelter_1.id,
+                            adoption_status: "Adoptable")
+
+        visit "/shelters/#{shelter_1.id}/pets"
+        expect(page).to have_content("#{pet_1.name}")
+
+        click_link "Delete"
+        expect(current_path).to eq("/pets")
+        expect(page).to_not have_content("#{pet_1.name}")
+    end
+
+    it "has links for shelter names" do
+        shelter_1 = Shelter.create( name: "4 Paws Rescue",
+                    address: "6567 W Long Dr.",
+                    city: "Littleton",
+                    state: "CO",
+                    zip: "80123")
+
+        pet_1 = Pet.create( name:    "Teddy",
+                            image: "https://i.pinimg.com/originals/47/4e/7d/474e7d2479512428a1a4716d5d1656eb.jpg",
+                            age:    "10",
+                            sex:   "M",
+                            description: "A sweet dog looking for a loving home",
+                            shelter_id: shelter_1.id,
+                            adoption_status: "Adoptable")
+        
+        visit("/pets")
+
+        click_link("#{pet_1.shelter.name}")
+        expect(current_path).to eq("/shelters/#{shelter_1.id}")
+    end
+
+    it "has links for pet names" do
+        shelter_1 = Shelter.create( name: "4 Paws Rescue",
+                    address: "6567 W Long Dr.",
+                    city: "Littleton",
+                    state: "CO",
+                    zip: "80123")
+
+        pet_1 = Pet.create( name:    "Teddy",
+                            image: "https://i.pinimg.com/originals/47/4e/7d/474e7d2479512428a1a4716d5d1656eb.jpg",
+                            age:    "10",
+                            sex:   "M",
+                            description: "A sweet dog looking for a loving home",
+                            shelter_id: shelter_1.id,
+                            adoption_status: "Adoptable")
+        visit("/pets")
+        click_link "#{pet_1.name}"
+
+        expect(current_path).to eq("/pets/#{pet_1.id}")
+
+        visit("/shelters/#{shelter_1.id}/pets")
+        click_link "#{pet_1.name}"
+
+        expect(current_path).to eq("/pets/#{pet_1.id}")
```

Comment:
Tests that clicking the pet name from the index page or the shelter pets page brings you to the pet's show page.